src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -300,8 +300,8 @@ const WorkspaceDashboard = _.flow(
   })
 
   const loadAzureStorage = withErrorReporting('Error loading Azure storage information.', async () => {
-    const { storageContainerName, location, sasUrl } = await Ajax(signal).AzureStorage.details(workspaceId)
-    setAzureStorage({ storageContainerName, storageLocation: location, sasUrl })
+    const { storageContainerName, location, sas } = await Ajax(signal).AzureStorage.details(workspaceId) || {}
```

Comment:
When would this request succeed and not return a value? It's called conditional on `azureContext`.

Reply:
hmm... It was needed originally because I did some de-structuring within the assignment, but I refactored that out, so the default value can go now.